turbo.json

```diff
@@ -3,7 +3,7 @@
 	"remoteCache": {
 		"signature": true
 	},
-	"globalEnv": [
+	"globalPassThroughEnv": [
```

Comment:
Is this correct? This will mean that these values won't contribute to cache keys, which I'm pretty sure we don't want (`CI_OS` should, at the very least)